draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -105,30 +105,30 @@ Selectively Disclosable JWT (SD-JWT)
 :  A JWT [@!RFC7515] created by the issuer, which is signed as a JWS [@!RFC7515],
    that supports selective disclosure as defined in this document.
 
-SD-JWT Salt/Value Container (SVC)
-:  A JSON object created by the issuer that contains mapping between
+Issuer-Issued Disclosures Object
+:  A JSON object created by the issuer that contains mapping between 
    raw claim values contained in the SD-JWT and the salts for each claim value.
 
-SD-JWT Release (SD-JWT-R)
-:  A JWT created by the holder that contains a subset of the claim values of an SD-JWT in a verifiable way.
+Holder-Selected Disclosures JWT
+:  A JWT created by the holder that contains a subset of the claim values of an SD-JWT in a verifiable way. 
```

Comment:
Holder-Selected Disclosures JWT (HS-Disclosures JWT)
:  A JWT created by the Holder that contains the Disclosures from an Issuer-Issued Disclosures Object that the Holder is disclosing to the Verifier. In addition to the Disclosures, it can contain other properties and may be signed by the Holder.